app/utils/filters.js

```diff
@@ -17,6 +18,14 @@ function validateFilters (filters) {
 
 function getFilters (req) {
   let filters = qs.parse(req.query)
+
+  // If a search is being performed without any filters specified, detault the from date filter to
+  // 30 days ago. We do this because the count query that is currently required by the search is
+  // very inefficient. If the search has been filtered, still allow the from date to be blank.
+  if (_.isEmpty(filters)) {
+    filters.fromDate = moment().tz('Europe/London').subtract(30, 'days').format('DD/MM/YYYY')
+  }
```

Comment:
Will this show up in the Front end?  I.e. will the user the `from date` being set?

Reply:
Yep
<img width="1436" alt="Screenshot 2020-06-24 at 13 12 10" src="https://user-images.githubusercontent.com/5648592/85553933-5afb9c80-b61c-11ea-81c9-055d9f27de35.png">